apps/communications/contacts/js/views/settings.js

```diff
@@ -350,58 +301,36 @@ contacts.Settings = (function() {
-      openSelectList(numFbContactsReq.result);
-    };
-
-    numFbContactsReq.onerror = function() {
-      openSelectList(0);
-      console.error('Number of fb contacts in device could not be retrieved',
-        numFbContactsReq.error && numFbContactsReq.error.name);
-    };
-
-    function openSelectList(numFilteredContacts) {
-      contacts.List.selectFromList(_('exportContactsAction'),
-        function onSelectedContacts(promise) {
-          // Resolve the promise, meanwhile show an overlay to
-          // warn the user of the ongoin operation, dismiss it
-          // once we have the result
-          requireOverlay(function _loaded() {
-            utils.overlay.show('preparing-contacts', null, 'spinner');
-            promise.onsuccess = function onSuccess(ids) {
-              // Once we start the export process we can exit from select mode
-              // This will have to evolve once export errors can be captured
-              contacts.List.exitSelectMode();
-              var exporter = new ContactsExporter(strategy);
-              exporter.init(ids, function onExporterReady() {
-                // Leave the contact exporter to deal with the overlay
-                exporter.start();
-              });
-            };
-            promise.onerror = function onError() {
-              contacts.List.exitSelectMode();
-              utils.overlay.hide();
-            };
-          });
-        },
-        null,
-        navigationHandler,
-        {
-          isDanger: false,
-          transitionLevel: EXPORT_TRANSITION_LEVEL,
-          filterList: [
-            {
-              'containerClass': 'disable-fb-items',
-              'numFilteredContacts': numFilteredContacts
-            }
-          ]
-        }
-      );
-    }
+    contacts.List.selectFromList(_('exportContactsAction'),
+      function onSelectedContacts(promise) {
+        // Resolve the promise, meanwhile show an overlay to
+        // warn the user of the ongoin operation, dismiss it
+        // once we have the result
+        requireOverlay(function _loaded() {
+          utils.overlay.show('preparing-contacts', null, 'spinner');
+          promise.onsuccess = function onSuccess(ids) {
+            // Once we start the export process we can exit from select mode
+            // This will have to evolve once export errors can be captured
+            contacts.List.exitSelectMode();
+            var exporter = new ContactsExporter(strategy);
+            exporter.init(ids, function onExporterReady() {
+              // Leave the contact exporter to deal with the overlay
+              exporter.start();
+            });
+          };
+          promise.onerror = function onError() {
+            contacts.List.exitSelectMode();
+            utils.overlay.hide();
+          };
+        });
+      },
+      null,
+      navigationHandler,
+      {
+        isDanger: false,
+        transitionLevel: EXPORT_TRANSITION_LEVEL
```

Comment:
At the end when migrating this must be transparent for the user, because the 'ContactsService' will not request FB, so no FB Contacts will appear in the list.